user_guide_src/source/changelogs/v4.7.0.rst

```diff
@@ -38,6 +38,10 @@ Enhancements
 Libraries
 =========
 
+- **CURLRequest:** Added ``dns_cache_timeout`` option to change default DNS cache timeout.
```

Comment:
In the future, please add new entries at the bottom rather than at the beginning.

Reply:
Ahh okayy😅I thought it would be alphabetical.

Reply:
I thought too it was alphabetical.

Reply:
Well, if two people are telling me it's supposed to be alphabetical, then I guess I'm wrong - sorry about that. Hopefully, I'll remember now.

Reply:
No problem, just an instinct based on what is available